### PR TITLE
Switch from travis encrypted files to vault. Closes #1296

### LINKED ISSUE #1296: Clean up Centaur-in-Travis build so someone other than Jeff can do it

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by mcovarr** [comment]
:+1: top-notch de-Jeff'ing.

File: src/bin/travis/resources/jes_centaur.conf.ctmpl [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{{with $refreshToken := vault (printf "secret/dsde/cromwell/common/refresh-token")}}
```
Comment (Horneth) [inline src/bin/travis/resources/jes_centaur.conf.ctmpl]: Does this file replace what's currently at `gs://cloud-cromwell-dev/travis-centaur/multiBackend.conf` ?

Comment (kshakir) [inline src/bin/travis/resources/jes_centaur.conf.ctmpl]: No. `jes_centaur.conf` replaces the `jes.conf` in `jesConf.tar.enc`.

Cromwell runs inception like, with an outer cromwell running an inner cromwell:
1. Outer cromwell uses `jes_centaur.conf`
2. Inner cromwell uses `mulitBackend.conf`

**Comment by Horneth** [comment]
👍

**Pull request merged**